PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -78,28 +78,14 @@ function configureToolboxSettings(settings) {
     var fileCollection = folder.SubFolders;
 
 
-    var maxMajor = 0,
-        maxMinor = 0,
-        maxPatch = 0,
-        maxVersionFolder = "";
+    var maxVersionFolder = "0";
     // Traverse through the fileCollection using the FOR loop
     // read the maximum version from toolbox filesystem
     for (var objEnum = new Enumerator(fileCollection); !objEnum.atEnd(); objEnum.moveNext()) {
         var folderObject = ( objEnum.item() );
         if (folderObject.Name.lastIndexOf('plugins') === -1) {
-            var versionMatch = /(\d+)\.(\d+)\.(\d+)/.exec(folderObject.Name),
-                major = parseInt(versionMatch[ 1 ]),
-                minor = parseInt(versionMatch[ 2 ]),
-                patch = parseInt(versionMatch[ 3 ]);
-            if (maxMajor === 0 || maxMajor <= major) {
-                maxMajor = major;
-                if (maxMinor === 0 || maxMinor <= minor) {
-                    maxMinor = minor;
-                    if (maxPatch === 0 || maxPatch <= patch) {
-                        maxPatch = patch;
-                        maxVersionFolder = folderObject.Name;
-                    }
-                }
+            if( compareVersion(folderObject.Name, maxVersionFolder) > 0 ){
```

Comment:
Please combine both IFs. Also please preserve coding style (see how you've formatted new and old IF).